app/src/main/java/com/gusta/wakemehome/AlarmAdapter.java

```diff
@@ -3,9 +3,11 @@
 import android.content.Context;
 import android.support.annotation.NonNull;
 import android.support.v7.widget.RecyclerView;
+import android.util.Log;
```

Comment:
As I understand it, the adapter is a "one-way street" - it only updates the UI according to the activity and model but not the other way. All presenting logic is supposed to be in activity, and than the adapter will show the updated element. see: https://android.jlelse.eu/android-handling-checkbox-state-in-recycler-views-71b03f237022.
I tried to find some examples and every adapter I saw had only the 3 basic methods. With that said - I am not sure that this is really the best practice, do you have any example or "best practices" about adapters with toggles?